cms/djangoapps/modulestore_migrator/api/__init__.py

```diff
@@ -0,0 +1,9 @@
+
+"""
+This is the public API for the modulestore_migrator.
+"""
+
+# These wildcard imports are okay because these api modules declare __all__.
+# pylint: disable=wildcard-import
+from .read_api import *
+from .write_api import *
```

Comment:
i've split up the API into these two parts to avoid a circular dependency in tasks.py

now:
* `api` depends on ->
* `api.write_api` depends on ->
* `tasks` depends on ->
* `api.read_api` depends on ->
* `models` depends on ->
* `data`

Reply:
Interesting. I had to deal with something like this as part of a last minute bugfix in another app (with a circular dependency based on tasks), and ended up doing a function-local import as a low-risk way to get around it without requiring refactoring.

A number of Django internal service patterns separate out reads vs. writes like this. I don't want to hijack this PR for the discussion, but I wonder if we should adopt this practice  more generally.